textproc/vislcg3/Portfile

```diff
@@ -25,14 +29,14 @@ long_description        Constraint Grammar (CG) is a methodological paradigm \
                         rule may be conditioned upon each other, negated or \
                         blocked by interfering words or tags.
 
-homepage                http://beta.visl.sdu.dk/constraint_grammar.html
-master_sites            http://beta.visl.sdu.dk/download/vislcg3/
+homepage                https://visl.sdu.dk/constraint_grammar.html
 
-checksums               md5    3875d7d36bedebe17317e32442c18b80 \
-                        sha1   8718fee3e8e9bb2edd58c11175b23661539309c1 \
-                        rmd160 fb0576a1f142bd47dfd35fb665408fe5dc9058e4
+checksums               rmd160 393f7cb49d40c265a1fcd29c13b999278712ee8d \
+                        sha256 ada9dd9f7cdd47d050f53fb87f5f4d590f899b7e5629e9fff694397a6b2bcd05 \
+                        size   357195
 
-depends_build           port:pkgconfig port:autoconf port:automake
-depends_lib             port:icu
+depends_build           port:cmake port:boost
+depends_lib             port:icu port:p5-getopt-long
```

Comment:
It is not correct to depend on a stub port like `p5-*`. Instead specify the perl version directly. I believe `p5.28-getopt-long` is preferred.

Reply:
You also need to use `depends_lib-append` here.

Reply:
It allows for consistent builds on the buildbots.